tests/resources/functions/README.md

```diff
@@ -0,0 +1,4 @@
+`php.tar.gz` includes dependency in `composer.json` and files:
+
+- `index.php` - Function that returns JSON with a `payload`, `variable` (from `customVariable`), and `unicode` message.
+- `timeout.php` - Simple JSON response `{pass:true}` but with 15 seconds sleep. Should be used to test timeout logic.
```

Comment:
Let's not keep tar files here. 
Instead keep the source files and build the tar files during the test like we do in Appwrite.

This way we always know the contents of the source code

Reference:
1. https://github.com/appwrite/appwrite/tree/master/tests/resources/functions
2.https://github.com/appwrite/appwrite/blob/master/tests/e2e/Services/Functions/FunctionsCustomClientTest.php#L215-L217
3.https://github.com/appwrite/appwrite/blob/master/tests/e2e/Services/Functions/FunctionsBase.php#L13

Reply:
Applied